PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -944,6 +964,43 @@ Bool_t AliEmcalJetTask::IsJetInPhos(Double_t eta, Double_t phi, Double_t r)
   }
   return kFALSE;
 }
+  
+/**
+ * Load the artificial tracking efficiency TF1 from a file into the member fTrackEfficiencyFunction
+ * @param path Path to the file containing the TF1
+ * @param name Name of the TF1
+ */
+void AliEmcalJetTask::LoadTrackEfficiencyFunction(const char* path, const char* name)
```

Comment:
I would suggest to use `const std::string &` here unless there is a compelling reason not to.

Reply:
Done. But really does it matter?

Reply:
Thanks. Both will work fine, but I would argue that if we're adding new code, we should use the standards when possible to make maintenance easier. `std::string` is the proper c++ string, so I think it is preferable to use it